pool-factory/src/storage.rs

```diff
@@ -4,6 +4,24 @@ use soroban_sdk::{contracttype, unwrap::UnwrapOptimized, Address, BytesN, Env, S
 pub(crate) const LEDGER_THRESHOLD: u32 = 518400; // TODO: Check on phase 1 max ledger entry bump
 pub(crate) const LEDGER_BUMP: u32 = 535670; // TODO: Check on phase 1 max ledger entry bump
 
+/// The configuration information about a reserve asset
+/// TODO: Duplicated from pool/storage.rs. Can this be moved to a common location?
+#[derive(Clone)]
+#[contracttype]
+pub struct ReserveConfig {
```

Comment:
this can be fetched from the Pool WASM, to avoid duplication. The Pool Factory is already dependent on it.